protos/dcs/hook/v0/hook.proto

```diff
@@ -163,3 +166,13 @@ message BanDetails {
   // When the ban will expire in unixtime
   uint64 banned_until = 6;
 }
+
+message GetUnitTypeRequest {
+  // The slot or unit ID of the unit to retrieve the type of
+  string id = 1;
```

Comment:
This is a number according to https://wiki.hoggitworld.com/view/DCS_func_getUnitType. Not sure if we should care, as it probably also seem to work with strings?

Reply:
Hmm, good question. It definitely works with strings as well, tried that yesterday.
I believe we even have to keep it as string since it works for slot IDs as well and those can have a `_2` suffix for multicrew aircraft, right (e.g. `4_2` for events for an F14 RIO).
Leaving it as string would also be consistent with the [`GetPlayerInfo.Slot`](https://github.com/DCS-gRPC/rust-server/blob/main/protos/dcs/net/v0/net.proto#L62) (which I based this of).

Reply:
The docs are a bit vague with `missionid`. Is this actually the slot ID? The unit ID? Can it be both? Did you try with both?

Reply:
I was asking myself the same thing, but yes, it can be both slot ID and unit ID.
I added a single F14 to the example mission provided by you (which gave me the slot ID "2"), then also tried with the E3 (unit ID "1").

Not sure why those are called differently (or even `missionid`), I would simply assume that the slot ID is (roughly) equal to a unit ID?

Anyways, it did work, yes 👍

Reply:
Cool. Thanks for the confirmation!